plugins/synaptics-rmi/synaptics-rmi.quirk

```diff
@@ -0,0 +1,4 @@
+[DeviceInstanceId=HIDRAW\VEN_06CB&DEV_76AE]
+Plugin = synaptics_rmi
+Vendor = Dell
+Name = XPS 13 Synaptics Touchpad
```

Comment:
Since commit 20cc9eebc57c6 I think we should try to avoid putting product names directly in quirk files so that they show up in the tree output exactly once.

Reply:
And actually if we use the generic strings I mentioned above, I don't think there is any need for a quirk file for this plugin.

Reply:
Vendor removed -- but don't we need the quirk file for the `Plugin = synaptics_rmi`?

Reply:
Hah, so I wasn't using `fu_plugin_add_rule (plugin, FU_PLUGIN_RULE_REQUIRES_QUIRK, FU_QUIRKS_PLUGIN);` -- which we certainly want to do I think. I'm not comfortable just issuing random ioctls on random devices.